srcpkgs/python3-saml2/template

```diff
@@ -1,15 +1,20 @@
 # Template file for 'python3-saml2'
 pkgname=python3-saml2
-version=5.0.0
-revision=2
+version=6.4.1
+revision=1
 wrksrc="pysaml2-${version}"
 build_style=python3-module
 hostmakedepends="python3-setuptools"
 depends="python3-cryptography python3-openssl python3-dateutil python3-pytz
  python3-requests python3-six python3-defusedxml"
+checkdepends="python3-pytest"
 short_desc="Python3 implementation of SAML Version 2 Standard"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="Apache-2.0"
 homepage="https://github.com/IdentityPython/pysaml2"
-distfiles="${PYPI_SITE}/p/pysaml2/pysaml2-${version}.tar.gz"
-checksum=deea75b252b5ead821f12ce01e9d88b0f6fbcde322c1595dcbb06f9660af0cc1
+distfiles="https://github.com/IdentityPython/pysaml2/archive/v${version}.tar.gz"
+checksum=341fe81e1fdff72e6082ba81645a9b75dab7482cf9b878f566d023311169f8c0
+
+do_check() {
+	pytest tests
+}
```

Comment:
Pytest has been added as an automatic option, so it should work without a custom `do_check`

Reply:
Thanks didn’t know that; I got to dig a little more to find why tests fail, expect it to be dep missing; on this one though will be using today as it needs itself as a dep to run test

Reply:
The automatic `pytest` option was just committed, so you will have to rebase to see it.

In general, you want to run `python3 -m pytest` rather than calling `pytest` directly. When run directly, the current working directory does not appear in the `PYTHONPATH`; when run as a module, the current working directory is the first entry in `PYTHONPATH`. This difference is enough to make a lot of tests run without further modification.

Otherwise, you can look at overriding PYTHONPATH or investigate some of the options to pytest; the automatic behavior uses `$make_check_args` and `$make_check_target` to allow you to customize to some extent without requiring you to define the function.

Reply:
I tried using pytest and the command ran as expected but like I thought, had to use tox as tests needs saml itself. Tests completed and this fixed the byte-compile during install I got earlier after I received an update

Reply:
Also xmlsec1-devel is needed as it seems to have xmlsec-openssl lib in it that tests failed on before adding the -devel package

Reply:
> The automatic `pytest` option was just committed, so you will have to rebase to see it.
> 
> In general, you want to run `python3 -m pytest` rather than calling `pytest` directly. When run directly, the current working directory does not appear in the `PYTHONPATH`; when run as a module, the current working directory is the first entry in `PYTHONPATH`. This difference is enough to make a lot of tests run without further modification.
> 
> Otherwise, you can look at overriding PYTHONPATH or investigate some of the options to pytest; the automatic behavior uses `$make_check_args` and `$make_check_target` to allow you to customize to some extent without requiring you to define the function.

Thanks for the explanation